src/components/Zap/index.js

```diff
@@ -5,15 +5,14 @@ function setContentForMetadata(userDetails) {
     let content = null;
     let lnurl = null;
     if ("lud06" in userDetails && userDetails.lud06.length > 0) {
-        content = '{"lud06": "' + userDetails.lud06 + '"}';
+        content = JSON.stringify({ lud06: userDetails.lud06 });
         lnurl = userDetails.lud06;
     } else if ("lud16" in userDetails && userDetails.lud16.length > 0) {
-        content = '{"lud16": "' + userDetails.lud16 + '"}';
+        content = JSON.stringify({ lud16: userDetails.lud16 });
```

Comment:
If I understood correctly what you wanted to achieve here, the `content` needs to be JSON object.
I.e. `{"foo": "bar"}`

Better use `JSON.stringify()` than trying to construct it manually in that case.

Reply:
This is a Deprecated file. I will remove it in the next PR.